metadeploy/api/models.py

```diff
@@ -650,7 +661,8 @@ class Job(HashIdMixin, models.Model):
     )
 
     def subscribable_by(self, user):
-        return self.is_public or user.is_staff or user == self.user
+        # TODO: Is this what we want?
+        return self.is_public or user.is_staff or user == self.user or not self.user
```

Comment:
@wlonk Currently an anonymous user can only `GET` a job if it's marked as `is_public=True`. This means that _any_ user can subscribe to a job WS channel. So we probably want to revert this, and add logic to `consumers.py` that instead checks if the session has a uuid for the org that started the job? (This could happen in another PR, as long as we note it.)